Update asciidoctorj-diagram dependency in order to use new plantuml.jar

The currently used plantuml version (plantuml.8053.jar) has a bad rendering algorithm. This jar is bundled in asciidoctorj-diagram.

For a new asciidoctorj-diagram version AsciidoctorJ 2.x is required, see [cannot load such file asciidoctor/logging · Issue #228 · asciidoctor/asciidoctor-diagram](https://github.com/asciidoctor/asciidoctor-diagram/issues/228). So this issue depends on #162 

As AsciidoctorJ 2.x has breaking changes it's not possible to just update the plugin dependency while using the plugin.

```Caused by: java.lang.IncompatibleClassChangeError: Found interface org.asciidoctor.ast.DocumentHeader, but class was expected
    at org.sahli.asciidoc.confluence.publisher.converter.AsciidocConfluencePage.pageTitle (AsciidocConfluencePage.java:173)
    at org.sahli.asciidoc.confluence.publisher.converter.AsciidocConfluencePage.newAsciidocConfluencePage (AsciidocConfluencePage.java:116)
    at org.sahli.asciidoc.confluence.publisher.converter.AsciidocConfluenceConverter.lambda$buildPageTree$0 (AsciidocConfluenceConverter.java:103)
    at java.util.ArrayList.forEach (ArrayList.java:1257)
    at org.sahli.asciidoc.confluence.publisher.converter.AsciidocConfluenceConverter.buildPageTree (AsciidocConfluenceConverter.java:98)
    at org.sahli.asciidoc.confluence.publisher.converter.AsciidocConfluenceConverter.convert (AsciidocConfluenceConverter.java:82)
    at org.sahli.asciidoc.confluence.publisher.maven.plugin.AsciidocConfluencePublisherMojo.execute (AsciidocConfluencePublisherMojo.java:108)

--- a/asciidoc-confluence-publisher-converter/src/test/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePageTest.java
+++ b/asciidoc-confluence-publisher-converter/src/test/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePageTest.java
@@ -958,7 +958,7 @@ public class AsciidocConfluencePageTest {
         AsciidocConfluencePage asciidocConfluencePage = newAsciidocConfluencePage(asciidocPage, UTF_8, TEMPLATES_FOLDER, assetsTargetFolderFor(asciidocPage));
 
         // assert
-        String expectedContent = "<ac:image ac:height=\"175\" ac:width=\"57\"><ri:attachment ri:filename=\"embedded-diagram.png\"></ri:attachment></ac:image>";
+        String expectedContent = "<ac:image ac:height=\"174\" ac:width=\"56\"><ri:attachment ri:filename=\"embedded-diagram.png\"></ri:attachment></ac:image>";
         assertThat(asciidocConfluencePage.content(), containsString(expectedContent));
         assertThat(exists(assetsTargetFolderFor(asciidocPage).resolve("embedded-diagram.png")), is(true));
     }
@@ -973,7 +973,7 @@ public class AsciidocConfluencePageTest {
         AsciidocConfluencePage asciidocConfluencePage = newAsciidocConfluencePage(asciidocPage, UTF_8, TEMPLATES_FOLDER, assetsTargetFolderFor(asciidocPage));
 
         // assert
-        String expectedContent = "<ac:image ac:height=\"175\" ac:width=\"57\"><ri:attachment ri:filename=\"included-diagram.png\"></ri:attachment></ac:image>";
+        String expectedContent = "<ac:image ac:height=\"174\" ac:width=\"56\"><ri:attachment ri:filename=\"included-diagram.png\"></ri:attachment></ac:image>";
         assertThat(asciidocConfluencePage.content(), containsString(expectedContent));
     }
 

--- a/pom.xml
+++ b/pom.xml
@@ -165,13 +165,13 @@
             <dependency>
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctorj</artifactId>
-                <version>1.5.4.1</version>
+                <version>1.5.8.1</version>
                 <scope>compile</scope>
             </dependency>
             <dependency>
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctorj-diagram</artifactId>
-                <version>1.5.4</version>
+                <version>1.5.18</version>
                 <scope>compile</scope>
             </dependency>
             <dependency>
@@ -206,7 +206,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-plugin-plugin</artifactId>
-                    <version>3.4</version>
+                    <version>3.6.0</version>
                 </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>

